content/en/docs/tasks/manage-daemon/pods-some-nodes.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Running Pods on Only Some Nodes
+content_type: task
+weight: 30
+min-kubernetes-server-version: 1.7
+---
+<!-- overview -->
+
+This page shows how you can run {{<glossary_tooltip term_id="pod" text="Pods">}} on only some {{<glossary_tooltip term_id="node" text="Nodes">}}
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
```suggestion
{{< include "task-tutorial-prereqs.md" >}}
This page assumes that you have a cluster with at least three worker nodes.
```

Skipping `{{< version-check >}}` because we can assume the reader is using Kubernetes much more recent than v1.7